book/src/basics/configuration.md

```diff
@@ -0,0 +1,38 @@
+# Configuration
+
+After running `solana-exporter generate`, a template config file will be created.
```

Comment:
The path needs to be specified here.

Please also explain the other subdirectories of `~/.solana-exporter` somewhere prominent. Caching needs to be explained, and the pages of those metrics that are persisted need to mention this fact.

Reply:
> The path needs to be specified here.

I wrote an explanation in `installation.md` for how `solana-exporter generate` works. I have reworded that section and added a short elaboration in `configuration.md`

> Please also explain the other subdirectories of `~/.solana-exporter` somewhere prominent.

AFAICT there's only the `config.toml` file and the persistent database.

> Caching needs to be explained, and the pages of those metrics that are persisted need to mention this fact.

I agree, let me add a section to explain how the persistent database is used. When I finish the "Exported Feeds" section I will be sure to note which gauges use the persistent database in their individual pages. Does that sound good?

Reply:
Yes, thank you!

Reply:
ea84de7 should help fix this, but I will need a bit more time to write up the rest of the feeds.